internal/objectstore/fileobjectstore_test.go

```diff
@@ -112,7 +112,7 @@ func (s *FileObjectStoreSuite) TestGetMetadataAndFileFoundWithIncorrectSize(c *g
 		Hash: hash,
 		Path: fileName,
 		Size: size + 1,
-	}, nil)
+	}, nil).Times(2)
```

Comment:
Maybe it's not possible, but can we have a test that tests when a file is indeed found on the file system correctly? In that case we would have something similar to this one but with `.Times(1)`